tezos_interop/tezos_interop.re

```diff
@@ -456,8 +461,11 @@ module Run_contract = {
         (command, [|command, file|]),
         Yojson.Safe.to_string(input_to_yojson(input)),
       );
+    print_endline(output);
     switch (Yojson.Safe.from_string(output) |> output_of_yojson) {
-    | Ok(data) => await(data)
+    | Ok(data) =>
+      print_endline("there");
```

Comment:
I think this might be a mistake

Reply:
Oh nice, forgot some high level debugging symbols